README.md

```diff
@@ -35,97 +28,48 @@ As a Maker
+  * Create the users table using the psql command: "CREATE TABLE users(id SERIAL PRIMARY KEY, email VARCHAR(60));"
 
-And/Or:
+  * Repeat steps 1-5 and name this database "chitter_test" (This will be our testing database) 
+  
 
-* Work on the CSS to make it look good.
+3. ### Testing 
 
-Good luck and let the chitter begin!
-
-Code Review
------------
-
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/main/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want at this moment.
-
-Automated Tests:
------
-
-Opening a pull request against this repository will will trigger Travis CI to perform a build of your application and run your full suite of RSpec tests. If any of your tests rely on a connection with your database - and they should - this is likely to cause a problem. The build of your application created by has no connection to the local database you will have created on your machine, so when your tests try to interact with it they'll be unable to do so and will fail.
-
-If you want a green tick against your pull request you'll need to configure Travis' build process by adding the necessary steps for creating your database to the `.travis.yml` file.
-
-- [Travis Basics](https://docs.travis-ci.com/user/tutorial/)
-- [Travis - Setting up Databases](https://docs.travis-ci.com/user/database-setup/)
-
-Notes on test coverage
-----------------------
-
-Please ensure you have the following **AT THE TOP** of your spec_helper.rb in order to have test coverage stats generated
-on your pull request:
-
-```ruby
-require 'simplecov'
-require 'simplecov-console'
-
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::Console,
-  # Want a nice code coverage website? Uncomment this next line!
-  # SimpleCov::Formatter::HTMLFormatter
-])
-SimpleCov.start
-```
+  * run the command 'rackup' to start the server 
 
-You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+  * visit localhost:9292 to checkout the app ! 
+  
+  
+  Further_features to add later:
+  ------------------------------------------------------------------------------------------------------------
+  
+  * Users can add their username, password and name to sign_up form 
+  * Would like to verify the email address is real
+  * Add the user that posts a tweet 
+  * Create login and log out function 
+  * Send emails when usernames are tagged in "cheet" 
+  * Use more HTML syntax and CSS to create a jazzy looking website 
```

Comment:
Great to see a README with information on the project, how to install gem dependancies, create the database, etc!